### PR TITLE
Update UFS for dust fix and remove nitrates by default 

### PR DESCRIPTION
**Pull request by bbakernoaa**
# Description
This PR updates the `ufs-weather-model` to the latest hash that included the dust fix [PR#1922](https://github.com/ufs-community/ufs-weather-model/pull/1922).  Along with this I removed the nitratrates by default in support of the GEFSv13 EP4 and EP5 experiments.  Removed unneeded inputs that should help with speed improvements. 

We will need to sync this file to the GOCART fix directories on all machines. `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/nexus/FENGSHA/FENGSHA_2022_NESDIS_inputs_10km_v3.2.nc`

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: parm/ufs/chem/ExtData.other
Comment (WalterKolczynski-NOAA) [inline parm/ufs/chem/ExtData.other:147]: Unless this is a temporary change, I'd prefer all these blocks of commented out fields to just be removed to keep things clean.

Comment (bbakernoaa) [reply parm/ufs/chem/ExtData.other:147]: I understand but I'm hesitant to remove them because future applications may use them.  @rmontuoro

Comment (JessicaMeixner-NOAA) [reply parm/ufs/chem/ExtData.other:147]: I echo @bbakernoaa and appreciate the desire to keep things cleaned and just simply remove, but from my understanding I think this might be a case where it'll help more people to leave it in.    While the configuration will be without nitrates for now, it seems very likely this is something that would be turned on for testing in the near future.  I'll of course defer to the aerosols group on this though in terms of how frequently they think this would be turned on/off and the need to leave this here.  It seems like the commented out diag table, and feild table would all fall under this.

Comment (bbakernoaa) [reply parm/ufs/chem/ExtData.other:147]: @JessicaMeixner-NOAA I concur. It would be helpful to leave it in for further development and testing rather than completely remove it.  Many contributors wouldn't know it was possible then.

File: parm/ufs/fv3/diag_table.aero
Comment (aerorahul) [inline parm/ufs/fv3/diag_table.aero:26]: Can we remove commented lines?  If they need to be re-enabled, they can be looked up in the git log to add back.

**Comment by JessicaMeixner-NOAA** [comment]
This PR needs: https://github.com/NOAA-EMC/global-workflow/issues/2005 before further testing. 

It looks like the settings here in this PR also need to be updated in tests in the ufs-weather-model as I believe the settings here are the ones desired, but don't quite match the ones in ufs-weather-model, which for example I believe still uses nitrates, etc.   This should not hold up this PR, but should be done to get ufs-weather-model and the global-workflow back in sync (or just simply in sync).

**Comment by JessicaMeixner-NOAA** [comment]
I have a test in the queue for this.

**Comment by JessicaMeixner-NOAA** [comment]
@bbakernoaa My forecast succeeded.  I think you can move this from draft to ready for review. 


The configuration updates need to get back to ufs-weather-model, but from what I can tell everything is as you want it here.

**Review by JessicaMeixner-NOAA — APPROVED** [review]

File: parm/ufs/chem/ExtData.other
Comment (WalterKolczynski-NOAA) [inline parm/ufs/chem/ExtData.other]: These need to be changed from the personal path to `ExtData/nexus/FENGSHA/...`

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
Looks good except the personal ExtData paths. Once those are corrected we can run CI.

**Comment by WalterKolczynski-NOAA** [comment]
Actually, I don't see a UFS update anymore. If that's no longer necessary, please update the title and description to match.

**Comment by JessicaMeixner-NOAA** [comment]
> Actually, I don't see a UFS update anymore. If that's no longer necessary, please update the title and description to match.

@WalterKolczynski-NOAA This PR should update UFS to https://github.com/ufs-community/ufs-weather-model/commit/63a43d9c1162e8284980f2ee51b525b56e576ecc from 10/26 to get the new GOCART updates unless a PR merged in before this updates to this or after.  @bbakernoaa isn't in today but  I assume it was unintentionally reverted merging in the latest develop at some point.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending successful completion of CI tests.

**Comment by WalterKolczynski-NOAA** [comment]
@bbakernoaa If this is ready, please mark it "Ready for Review" and I will start the CI.

**Comment by TerrenceMcGuinness-NOAA** [comment]
Just restarted your CI runs because I had just brought the CI driver up from some output updates.

**Pull request merged**
